src/broker/content-cache.c

```diff
@@ -953,6 +976,15 @@ struct content_cache *content_cache_create (flux_t *h, attr_t *attrs)
     if (!(cache->f_sync = flux_sync_create (h, 0))
         || flux_future_then (cache->f_sync, sync_max, sync_cb, cache) < 0)
         goto error;
+    if (!(cache->w = flux_timer_watcher_create (
+              flux_get_reactor(h),
+              1,
+              1,
+              timer_cb,
+              cache)))
+        goto error;
+    flux_watcher_start (cache->w);
+
```

Comment:
Currently this timer fires unconditionally (whether or not we are collecting stats).  On a compute node, this might be stealing cycles from a job, or evicting cache lines, etc.. Is there a way we can disable the timer if we are not collecting stats?

I think it would be OK to piggy back on the `sync_cb()` which gets called every heartbeat (default T=2s).  See also: [flux_sync_create(3)](https://flux-framework.readthedocs.io/projects/flux-core/en/latest/man3/flux_sync_create.html).  Maybe just change the `timer_cb()` function into one that gets called from `sync_cb()`.